task/sanity-inspect-image/0.1/README.md

```diff
@@ -0,0 +1,23 @@
+# sanity-inspect-image task
+
+## Description:
+Use Skopeo to inspect and analyze manifest data from the target source image of a container.
```

Comment:
Maybe we can also mention that if the image has a direct base image, this task also attempts to inspects that image.

Reply:
Fixed in most recent patch.